include/mbgl/style/expression/coercion.hpp

```diff
@@ -23,6 +23,8 @@ class Coercion : public Expression  {
 
     EvaluationResult evaluate(const EvaluationContext& params) const override;
     void eachChild(const std::function<void(const Expression&)>& visit) const override;
+    
```

Comment:
The `Coercion` class-level comment at L14 is missing information about coercion of simple text to formatted text expressions.

Reply:
🤔 Actually I think that header comment may be outdated? It implies `Coercion` is just for `to-number` and `to-color`, which is no longer true. But it does make me wonder whether the default `Coercion` behavior of serial evaluation should really be supported for the `format` coercion -- since I don't think there's even any way to use it (the only way to create this coercion is through the implicit coerce on the `text-field` property).

Reply:
I moved the comment and disabled the coercion for `type::Formatted` (although this is a divergence from gl-js, there shouldn't actually be a way to exercise this code path either way, and this seems like a clearer expression of intent).